README.md

```diff
@@ -55,6 +55,149 @@ Edit the file `cdc-accelerator.json` in the project directory and add the `sourc
 }
 ```
 
+### How to configure the use of features on the file cdc-accelerator.json
+
+On the cdc-accelerator.json file, there are two mandatory properties that the user has to fill, the "source" and "deploy".
```

Comment:
Use the same syntax to refer to files, `cdc-accelerator.json` or other names, `source`, `deploy` (as it is in the previous paragraph)